### ISSUE TITLE
move_cursor_to doesn't update screen_x and screen_y

### ISSUE THREAD

**Issue by abadonna** [subsoap/defos#77]
1. set new cursor position with move_cursor_to 
2. initiate any input action - action.x\action.y are correct, but action.screen_x\screen_y have old values

**Comment by dapetcu21** [comment]
Hi! On what platform?

**Comment by abadonna** [comment]
osx running from editor

**Comment by abadonna** [comment]
Any news?

**Comment by subsoap** [comment]
I don't think we can update screen_x/screen_y that's an engine thing. @britzl what do you think?

**Comment by abadonna** [comment]
It would be nice to force input event somehow

**Comment by subsoap** [comment]
We've requested to be able to simulate input before. You should post on the Defold forums asking again so it gets more attention.
